### PR TITLE
PR: Fix/bind custom value set operation outcome.issue.detail

### PR DESCRIPTION
**Pull request by alexzautke**

### CODE REVIEW

File: Vonk.Plugin.DocumentOperation/DocumentService.cs [hunk 1]
```diff
@@ -269,12 +269,12 @@ private IssueComponent ReferenceNotResolvedIssue(string failedReference, bool mi
             if (missingReferenceIsLocal)
             {
                 issue.Code = IssueType.NotFound;
-                issue.Details = new CodeableConcept("http://hl7.org/fhir/ValueSet/operation-outcome", "MSG_LOCAL_FAIL", "Unable to resolve local reference to resource " + failedReference);
+                issue.Details = new CodeableConcept("http://vonk.fire.ly/fhir/ValueSet/OperationOutcomeIssueDetails", "MSG_LOCAL_FAIL", "Unable to resolve local reference to resource " + failedReference);
```
Comment (marcovisserFurore) [inline Vonk.Plugin.DocumentOperation/DocumentService.cs]: MSG_LOCAL_FAIL is part of the "normal" valueset of operation-outcome, right? So we can just use that code from that system, or not?

File: Vonk.Plugin.DocumentOperation/DocumentService.cs [hunk 2]
```diff
@@ -269,12 +269,12 @@ private IssueComponent ReferenceNotResolvedIssue(string failedReference, bool mi
...
             }
             else
             {
                 issue.Code = IssueType.NotSupported;
-                issue.Details = new CodeableConcept("http://hl7.org/fhir/ValueSet/operation-outcome", "MSG_EXTERNAL_FAIL", "Resolving external resource references (" + failedReference + ") is not supported");
+                issue.Details = new CodeableConcept("http://vonk.fire.ly/fhir/ValueSet/OperationOutcomeIssueDetails", "MSG_EXTERNAL_FAIL", "Resolving external resource references (" + failedReference + ") is not supported");
```
Comment (marcovisserFurore) [inline Vonk.Plugin.DocumentOperation/DocumentService.cs]: I would use the $"{variable}"  construction

```
issue.Details = new CodeableConcept("http://vonk.fire.ly/fhir/ValueSet/OperationOutcomeIssueDetails", "MSG_EXTERNAL_FAIL", $"Resolving external resource references ({failedReference}) is not supported");
```

File: Vonk.Plugin.DocumentOperation/DocumentService.cs (hunk 1 above)
Comment (alexzautke) [reply Vonk.Plugin.DocumentOperation/DocumentService.cs]: Resolved by agreeing that we stick to a single ValueSet.

**Comment by alexzautke** [comment]
Addressed review remarks. Approved?

**Pull request merged**